package.json

```diff
@@ -17,8 +17,7 @@
     "schema-utils": "^2.6.5"
   },
   "peerDependencies": {
-    "@babel/core": "^7.0.0",
-    "webpack": ">=2"
+    "@babel/core": "^7.0.0"
```

Comment:
I believe specifying `webpack` peerDependencies is a de-facto convention of webpack plugins.

css-loader: https://github.com/webpack-contrib/css-loader/blob/master/package.json#L43
file-loader: https://github.com/webpack-contrib/file-loader/blob/master/package.json#L41
etc.

Reply:
I use a storybook that needs `babel-loader`, and storybook uses its `webpack` version.
I don't need a `webpack` in my repo, there is only a babel build there.

Maybe it's storybook to add `webpack` to peerDependencies then?

Reply:
> Maybe it's storybook to add webpack to peerDependencies then?

Correct.